src/peerbook/libp2p_peer.proto

```diff
@@ -19,6 +19,7 @@ message peer {
 message signed_peer {
     peer peer = 1;
     bytes signature = 2;
+    map<string, bytes> metadata = 3;
```

Comment:
is this not stripped by encode now?

Reply:
no.. it's only stripped by encode_list. We rely on this encode for peerbook storiage